src/v2/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -96,7 +105,9 @@ export class TransactionDetailsSummaryItem extends React.Component<
       <>
         <Entry
           label={isBuyerOffer ? "Your offer" : "Seller's offer"}
-          value={offerOverride || (offer && offer.amount) || emDash}
+          value={
+            offerOverride || (offer && offer.amount) || this.amountPlaceholder
```

Comment:
Is the "to be confirmed" copy going to be rendered on the price field in some scenarios? I wonder if the review step would always have a price.

Reply:
The review step always has a price - if and only shipping and tax are available. So it should only render `to be confirmed` for inquiry offers with missing metadata.